src/udcutils.pas

```diff
@@ -360,7 +366,37 @@ function mbExpandFileName(const sFileName: String): String;
   end;
 end;
 
-function cnvFormatFileSize(iSize: int64; FSF: TFileSizeFormat; Number: integer): string;
+function IntToStrTS(const APositiveValue: Int64): String;
+var i, vSrcLen, vSrcI, vSrcNumberNo, vResLen: byte;
+begin
+  if APositiveValue < 0 then
+    raise Exception.Create('IntToStrTS gets only positive values!');
```

Comment:
I think it is better to use `Exit(IntToStr(APositiveValue))` than crash.

Reply:
I think to better find the error but not to hide. But OK.